queries/models/validacao_dados_jae/CHANGELOG.md

```
@@ -1,5 +1,17 @@
 # Changelog - validacao_dados_jae
 
+## [3.0.0] - 2025-08-19
+
+
+# Adicionado
+
+- Cria nova lógica para o cálculo de integrações não realizada (https://github.com/prefeitura-rio/pipelines_rj_smtr/pull/793):
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)